plugins/registry/functions.go

```diff
@@ -135,6 +159,17 @@ func dockerTag(imageID string, imageTag string) bool {
 	return true
 }
 
+func dockerRemoveTag(image string) bool {
```

Comment:
This could all be replaced by `common.RemoveImages()`: https://github.com/dokku/dokku/blob/master/plugins/common/docker.go#L339

Reply:
let me have a look